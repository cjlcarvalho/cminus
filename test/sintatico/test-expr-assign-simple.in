void main(void)
{
    x = y;
    x = 10;
    x = (1 + 2) * y;
}
