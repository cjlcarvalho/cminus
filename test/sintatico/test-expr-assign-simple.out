[program 
  [decfunc 
    [void]
    [main]
    [paramlist]
    [block 
      [assign [var [x]] [var [y]]] 
      [assign [var [x]] [10]] 
      [assign [var [x]] 
        [* 
          [+ [1] [2]] [var [y]]]] 
    ]
  ]
]
